rust/README.md

```diff
@@ -102,3 +111,15 @@ frontend](./agama-cli/doc/backend-for-testing.md)*.
 ## Caveats
 
 * If no product is selected, the `probe` command fails.
+
+## Packaging
+
+A packaging files lives in `package` directory. Agama follows
+[rust packaging guidelines](https://en.opensuse.org/openSUSE:Packaging_Rust_Software).
+For testing changes to spec file use simple `osc branch YaST:Head:Agama agama-cli`
+and copy modified spec file to that branch.
+If it needs also specific code from git branch, then modify `_service` file and
+change git branch name in `<revision>` tag. Then run `osc service runall`.
+Note: for leap `cargo_audit` does not work with older python, so comment out that
```

Comment:
```suggestion
Note: for openSUSE Leap `cargo_audit` does not work with older Python, so comment out that
```

Reply:
Oh, _this_ is the part that I hit yesterday :facepalm: . I should read READMEs more often.